CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
-0.4.1 (Next)
+0.4.2 (Next)
 ============
 
 * Your contribution here.
 
+0.4.1 (10/23/2015)
```

Comment:
This one is still wrong. It should just be 0.4.1 (next), you're not doing a release ;)